src/web/app/pages-help/instructor-help-page/instructor-help-page.component.ts

```diff
@@ -4,6 +4,8 @@ import { ActivatedRoute, Params } from '@angular/router';
 import { PageScrollService } from 'ngx-page-scroll-core';
 import { environment } from '../../../environments/environment';
 import { InstructorHelpCoursesSectionComponent } from './instructor-help-courses-section/instructor-help-courses-section.component';
+import { InstructorHelpGeneralSectionComponent } from './instructor-help-general-section/instructor-help-general-section.component';
+// General import
```

Comment:
What is this comment about?

Reply:
I had that comment there so I could remember where to put the general section import as it needed to be alphabetical while i tried some different things. I can remove it.